### PR TITLE
docs: update the readme to latest working one

### PR DESCRIPTION
**Pull request by guagualvcha**
### Description
The current readme is totally out of date, this PR fix it.

### Rationale
The following update is made in this PR:

1. Dex-related chapter is removed.
2. Fix the not working CLI tutorial.
3. Adding more futures.

### Example
No

### Changes
Document fix.

### Preflight checks

- [x] build passed (`make build`)
- [x] tests passed (`make test`)
- [x] integration tests passed (`make integration_test`)
- [x] manual transaction test passed (cli invoke)

### CODE REVIEW

**Review by yutianwu — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -13,10 +13,11 @@ Beacon Chain has the basic features of most blockchains:
 - Issuing new digital assets (we have a standard called BEP-2)
 - Mint/burn, freeze/unfreeze, lock/unlock of digital assets
 
-It has DEX and trading-specific functionality:
-- Propose exchange listing for trading pairs
-- Creating maker/taker orders for traders
-- Listing assets from other chains using atomic swaps (BEP-3)
+Besides it get many other rich features:
```
Comment (forcodedancing) [inline README.md:16]: `get` to `gets` or `has` ?

File: README.md [hunk 2]
```diff
@@ -25,70 +26,67 @@ It has DEX and trading-specific functionality:
 * Full nodes are not validator nodes, but anyone can get a copy of the whole blockchain and validate it.
 * No PoW means block times are very fast.
 * UXTO/account does not matter as we just use the [cosmos](https://github.com/cosmos/cosmos-sdk/tree/master/x/bank) bank.
-* Features like the DEX will run directly on the node as apps written in golang.
+* Features like the DEX will run directly on the node as apps written in golang(deprecated now).
```
Comment (forcodedancing) [inline README.md:29]: `golang(deprecated now)` -> `golang (deprecated now)`

Comment (guagualvcha) [reply README.md:29]: fixed

File: README.md [hunk 3]
```diff
@@ -25,70 +26,66 @@ It has DEX and trading-specific functionality:
...
 
 ## Getting Started
 
 ### Environment setup
 
-If you do not have golang yet, please [install it](https://golang.org/dl) or use brew on macOS: `brew install go` and `brew install dep`.
+#### Requirement
+Go version above 1.17 is required.
 
+Please [install it](https://golang.org/dl) or use brew on macOS: `brew install go` and `brew install dep`.
```
Comment (owen-reorg) [inline README.md:39]: `dep` should be unnecessary now

Comment (owen-reorg) [inline README.md:39]:
```suggestion
Please [install it](https://go.dev/doc/install) or use brew on macOS: `brew install go`.
```

File: README.md [hunk 4]
```diff
@@ -102,7 +99,9 @@ Please refer to the document for joining [mainnet](https://docs.bnbchain.world/d
 Assets may be issued through `bnbcli` while the blockchain is running; see here for an example:
 
 ```bash
-$ bnbcli tokens issue bnb -n bnb -s 100000
+$ chainId=`cat ~/.bnbchaind/config/genesis.json  |jq .chain_id --raw-output`
```
Comment (owen-reorg) [inline README.md:102]:
```suggestion
$ chainId=`cat ~/.bnbchaind/config/genesis.json | jq .chain_id --raw-output`
```

File: README.md [hunk 5]
```diff
@@ -112,39 +111,16 @@ This will post a transaction with an `IssueMsg` to the blockchain, which contain
...
-      },
-      {
-         "symbol" : "XYZ",
-         "free" : "0.98999900",
-         "locked" : "0.00000100",
-         "frozen" : "0.00000000"
-      }
-   ]
-}
+$  ./build/bnbcli account bnb1elavnu4uyzt43hw380vhw0m2zl7djz0xeac60t  --chain-id ${chainId} |jq
```
Comment (owen-reorg) [inline README.md:123]:
```suggestion
$  ./build/bnbcli account bnb1elavnu4uyzt43hw380vhw0m2zl7djz0xeac60t --chain-id ${chainId} | jq
```

File: README.md [hunk 6]
```diff
@@ -154,30 +130,31 @@ You have to send a transaction to send assets to another address, which is possi
...
-$ bnbcli tx 492B08FFE364D389BB508FD3507BBACD3DB58A98
+$  ./build/bnbcli tx 40C2B27B056A63DFE5BCE32709F160F53633C0EBEBBD05E1AC26419D35303765 --chain-id ${chainId} 
 ```
 
-Then you can check the balance of pepe's key to see that he now has 1000 satoshi units of `mycoin`:
+Then you can check the balance of pepe's key to see that he now has 100 satoshi units of `FBTC-4C9`:
 
 ```bash
-$ bnbcli account B71E119324558ABA3AE3F5BC854F1225132465A0
+$ ./build/bnbcli account bnb1u5mvgkqt9rmj4fut60rnpqfv0a865pwnn90v9q  --chain-id ${chainId} |jq
```
Comment (owen-reorg) [inline README.md:148]:
```suggestion
$ ./build/bnbcli account bnb1u5mvgkqt9rmj4fut60rnpqfv0a865pwnn90v9q  --chain-id ${chainId} | jq
```

File: README.md (hunk 3 above)
Comment (guagualvcha) [reply README.md:39]: fixed

File: README.md (hunk 1 above)
Comment (guagualvcha) [reply README.md]: fixed, change to `has`

**Review by yutianwu — APPROVED** [review]

**Pull request merged**